DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/TutorialVideoViewModelTests.cs

```diff
@@ -126,58 +127,16 @@ public void TutorialVideo_should_have_courseTitle()
             tutorialVideoViewModel.CourseTitle.Should().BeEquivalentTo(courseTitle);
         }
 
-        [Test]
-        public void TutorialVideo_should_parse_absolute_url()
-        {
-            // Given
-            const string videoPath = "https://example.com/testVideo.mp4";
-            var expectedTutorialVideo = TutorialContentHelper.CreateDefaultTutorialVideo(
-                videoPath: videoPath
-            );
-
-            // When
-            var tutorialVideoViewModel = new TutorialVideoViewModel(
-                config,
-                expectedTutorialVideo,
-                CustomisationId,
-                SectionId,
-                TutorialId
-            );
-
-            // Then
-            tutorialVideoViewModel.VideoPath.Should().Be(videoPath);
-        }
-
-        [Test]
-        public void TutorialVideo_should_parse_protocol_relative_url()
-        {
-            // Given
-            const string videoPath = "example.com/testVideo.mp4";
-            var expectedTutorialVideo = TutorialContentHelper.CreateDefaultTutorialVideo(
-                videoPath: videoPath
-            );
-
-            // When
-            var tutorialVideoViewModel = new TutorialVideoViewModel(
-                config,
-                expectedTutorialVideo,
-                CustomisationId,
-                SectionId,
-                TutorialId
-            );
-
-            // Then
-            tutorialVideoViewModel.VideoPath.Should().Be($"https://{videoPath}");
-        }
-
-        [Test]
-        public void TutorialVideo_should_parse_relative_path()
+        [TestCase("https://example.com/testVideo.mp4")]
+        [TestCase("example.com/testVideo.mp4")]
+        [TestCase("/testVideo.mp4")]
+        public void TutorialVideo_should_parse_path(string videoPath)
         {
             // Given
-            const string videoPath = "/testVideo.mp4";
             var expectedTutorialVideo = TutorialContentHelper.CreateDefaultTutorialVideo(
                 videoPath: videoPath
             );
+            var expectedParsedPath = ContentUrlHelper.GetContentPath(config, videoPath);

```

Comment:
I'd rather this test didn't rely on `GetContentPath` 🙂

Reply:
Yep, changed in ca246da 🙂